program/c/src/oracle/upd_aggregate.h

```diff
@@ -222,7 +223,9 @@ static inline bool upd_aggregate( pc_price_t *ptr, uint64_t slot, int64_t timest
   ptr->agg_.price_  = agg_price;
   ptr->agg_.conf_   = (uint64_t)agg_conf;
 
-  upd_twap( ptr, agg_diff );
+  if ( update_twap ) {
+    upd_twap( ptr, agg_diff );
+  }
   return true;
```

Comment:
This return true triggers and accumulator message and a PriceCumulative update, you need to be careful about that